src/evaluator/evaluator.js

```diff
@@ -38,3 +38,17 @@ function getNativeMasterSize(apkName, flavorToBuild, buildPath) {
     typeof sizeOp === `string` ? sizeOp.trim().split(/\s+/)[0] : 0;
   return apkSize;
 }
+
+export function getRNBundleMasterSize(flavorToBuild, bundlePath) {
+  console.log("inside get bundle size method")
+  const bundleName = "index.android.bundle"
+  const bundleFlavor = getBundleFlavor(flavorToBuild)
+  console.log("Bundle flavor :: ", bundleFlavor)
+  execSync(`yarn bundle:${bundleFlavor}:android`, { encoding: "utf-8" });
+  const sizeOp = execSync(`cd ${bundlePath} && du -k ${bundleName}`, {
+    encoding: "utf-8",
+  });
+  console.log(sizeOp);
+  const bundleSize = typeof sizeOp === `string` ? sizeOp.trim().split(/\s+/)[0] : 0;
+  return bundleSize;
```

Comment:
take bundle command in this or directory as this method can change and will require change in the action